scripts/west_commands/genboard/config.yml

```diff
@@ -0,0 +1,93 @@
+products:
```

Comment:
I know we don't have all the info needed, such as ram, flash, and SoC package name, but I think it would be much better if available SoCs could be obtained from the Nordic's soc.yaml and not duplicated into this file.

Especially because I fear we will forget to pay attention to this file, as we are not creating new Nordic boards using this tooling.